StRoot/StEvent/StFwdTrackCollection.h

```diff
@@ -19,7 +19,7 @@ class StFwdTrack;
 class StFwdTrackCollection : public StObject {
 public:
     // StFwdTrackCollection();
```

Comment:
I believe we prefer to have all of our classes default constructable.  Other than that, this looks good to me.

Reply:
updated in 6d24ade